onnx/inliner/inliner.cc

```diff
@@ -720,5 +722,85 @@ void InlineSelectedFunctions(
   }
   InlinerImpl::InlineSelectedFunctions(model, to_inline, schema_registry);
 }
+
+// Helper class to access InliningRenamer functionality
+class RenamerHelper : public InliningRenamer {
```

Comment:
Can you move these changes into InliningRenamer directly, and avoid this new class? (Change method access to public for needed methods, and add any needed new method.)